client/java/src/main/java/com/linecorp/centraldogma/client/updater/CentralDogmaBeanFactory.java

```diff
@@ -61,8 +63,21 @@
  * >     public void setA(int a) { this.a = a; }
  * >     public String getB() { return b; }
  * >     public void setB(String b) { this.b = b; }
+ * >     public Revision getRevision() { return null; }
  * > }
  * }</pre>
+ *
+ * <p>In the following example, callback will be called when a property is updated with a new value
+ *
+ * <pre>{@code
+ * > CentralDogmaFactory factory = new CentralDogmaFactory(dogma, new ObjectMapper());
```

Comment:
You don't have to use `>` here and following lines. It is used when the line starts with `@`